src/test/java/seedu/address/logic/commands/ExportCommandTest.java

```diff
@@ -65,28 +65,28 @@ public void equals() {
     public void execute_validInput_fileCreated() {
         String expectedMessage = String.format(
                 ExportCommand.MESSAGE_EXPORT_SUCCESS,
-                firstFilePath
+                firstDocumentPath.toAbsolutePathString()
         );
 
-        deleteFileIfExists(firstFilePath);
+        deleteFileIfExists(firstDocumentPath);
         assertCommandSuccess(firstCommand, model, expectedMessage, model);
-        assertTrue(isFilePresent(firstFilePath));
-        deleteFileIfExists(firstFilePath);
+        assertTrue(isFilePresent(firstDocumentPath));
+        deleteFileIfExists(firstDocumentPath);
     }
 
-    private boolean isFilePresent(FilePath filePath) {
-        File file = new File(filePath.toString());
+    private boolean isFilePresent(DocumentPath documentPath) {
+        File file = new File(documentPath.toString());
         return file.exists();
     }
 
-    private void deleteFile(FilePath filePath) {
-        File file = new File(filePath.toString());
+    private void deleteFile(DocumentPath documentPath) {
+        File file = new File(documentPath.toString());
         file.delete();
     }
 
-    private void deleteFileIfExists(FilePath filePath) {
-        if (isFilePresent(filePath)) {
-            deleteFile(filePath);
+    private void deleteFileIfExists(DocumentPath documentPath) {
+        if (isFilePresent(documentPath)) {
+            deleteFile(documentPath);
```

Comment:
You may want to consider using try-catch statements if `!isFilePresent` is an exceptional case. But this is very debateable; here's an interesting read - https://stackoverflow.com/questions/8621762/java-if-vs-try-catch-overhead